subxt/src/backend/unstable/mod.rs

```diff
@@ -350,6 +388,8 @@ impl<T: Config + Send + Sync + 'static> Backend<T> for UnstableBackend<T> {
     async fn stream_runtime_version(&self) -> Result<StreamOfResults<RuntimeVersion>, Error> {
         // Keep track of runtime details announced in new blocks, and then when blocks
         // are finalized, find the latest of these that has runtime details, and clear the rest.
+        //
+        // The backend driver already re-starts the subscription if gets re-connected.
```

Comment:
If we are disconnected long enough, can we miss a runtime version entirely? I'm not sure whether I care too much about this though!

Reply:
yes we can